fluentui_persona/src/main/java/com/microsoft/fluentui/tokenized/persona/Utils.kt

```diff
@@ -30,7 +31,7 @@ class Person(
     val lastName: String = "",
     val email: String? = null,
     @DrawableRes val image: Int? = null,
-    val imageBitmap: @RawValue ImageBitmap? = null,
+    val imageBitmap: @RawValue Bitmap? = null,
```

Comment:
Lets rename this as bitmapImage to be more clear on the Type. Do mention this as API break as well during release.

Reply:
Okay, will mention